app/upgrades/v2.2.0/upgrade.go

```diff
@@ -56,7 +64,51 @@ func ExecuteProposal(ctx sdk.Context, dk distribution.Keeper, bk bank.Keeper, ak
 	)
 	ak.SetAccount(ctx, pva)
 
-	return dk.DistributeFromFeePool(ctx, amount, vestingAcc)
+	if err := dk.DistributeFromFeePool(ctx, amount, vestingAcc); err != nil {
+		return err
+	}
+
+	// first clear old data
+	ck.ClearInitialClaimables(ctx)
+
+	// add/update the newly added claim records and update module account balance
+	var amount sdk.Coins
+	for _, record := range NewClaimRecords {
+		amount = amount.Add(record.ClaimableAmount...)
+
+		// update the claim record in claim module
+		if err := ck.UpdateClaimRecord(ctx, *record); err != nil {
+			return err
+		}
+	}
+	ctx.Logger().Info(fmt.Sprintf("added new claim records: %d", len(NewClaimRecords)))
+
+	// get airdrop account
+	airdropAccAddr, err := sdk.AccAddressFromBech32("pasg1lel0s624jr9zsz4ml6yv9e5r4uzukfs7hwh22w")
+	if err != nil {
+		return err
+	}
+
+	oldAmount := sdk.Coins{
+		sdk.NewCoin(amount[0].Denom, sdk.NewInt(18946800000000)),
+	}
+
+	amount = amount.Sub(oldAmount)
+
+	// send the added balances from airdrop account to claim module account
+	if err := bk.SendCoinsFromAccountToModule(ctx, airdropAccAddr, claimtypes.ModuleName, amount); err != nil {
+		return err
+	}
+	ctx.Logger().Info(fmt.Sprintf("sent coins: %s from airdrop account to claim module account", amount.String()))
+
+	params := ck.GetParams(ctx)
+	params.AirdropEnabled = true
+	params.AirdropStartTime = time.Date(2023, 10, 12, 15, 0, 0, 0, time.UTC) // (dd/mm/yyyy: 16/10/2023, 15:00)
```

Comment:
which one is correct(comment  or airdrop-start-time)?

Reply:
@anilCSE mentioned in my old PR the actual date should be from 23/10/2023.